src/views/HomeView.vue

```diff
@@ -72,6 +86,16 @@ onMounted(() => {
     </div>
     <FooterComponent></FooterComponent>
   </main>
+  <div v-if="isRaining" class="fade-in" @animationend="stopRainingEffect"></div>
 </template>
 
-<style lang="scss"></style>
+<style lang="scss">
```

Comment:
Are we using Scss? Copy/paste is okay if it get cleaned up. I feel like this has been left behind.